### ISSUE TITLE
Наименования классов

### ISSUE THREAD

**Issue by xYor** [callmevladik/test#3]
http://joxi.ru/BA0O3lQiJ51Jn2
не совсем корректно.
1 - вообщем по проекту нужно придерживатся 1 стиля написания кода (в данном случае используются и "-" и "_".
2 - советую прочитать про БЭМ (https://ru.bem.info/methodology/quick-start/). Это просто методология наименования классов, никаких ни плагинов ничего.

**Comment by callmevladik** [comment]
Да, про бэм слышал и читал немного, но всё равно толком не использовал, переделаю
